### ISSUE TITLE
Prevent minimizing the window while retaining the ability to resize & maximize?

### ISSUE THREAD

**Issue by DaveInCaz** [fluentribbon/Fluent.Ribbon#842]
I have been looking into ways to hide or disable the ability to minimize a RibbonWindow. In my application, this window is a modal child of the main application, so minimizing only this window would not make sense. However, I do need to retain the ability to resize and maximize / restore it.

I read a partial solution in another post:
https://github.com/fluentribbon/Fluent.Ribbon/issues/426#issuecomment-295814358

but that approach would allow the window's context menu to still be active.

I've also tried out using Win32 calls (based on https://stackoverflow.com/a/958980/3195477) which does not work, I think because the RibbonWindow title bar is not the Win32 window's title bar.

Are there any other options I should consider? 

 * For example, if I changed from using RibbonWindow to a standard Window, what are the consequences (if this is described elsewhere I apologize that I haven't found it yet).

 * Or, perhaps, is there a way to control the contents of the RibbonWindow context menu?

Thanks!

---
### Environment

- Fluent.Ribbon __v7.10__
- Windows __10__
- .NET Framework __4.72__

**Comment by batzen** [comment]
There is currently no way to prevent only minimizing out of the box.
You could set `EnableMinimize` to `false` on the `WindowChromeBehavior` (which is attached to `RibbonWindow`) and grab `PART_Min` from `WindowCommands` to set its visibility.

**Comment by DaveInCaz** [comment]
Based on the suggestion above I made a `CustomWindowCommands` class which overrides `OnApplyTemplate()`, found `PART_Min`, etc. and that worked great, thank you.

------

Initially I tried to use that class from XAML, like this:

```
<Fluent:RibbonWindow
    x:Class="MyWindow"
   ....
   >

   <Fluent:RibbonWindow.WindowCommands>
       <local:CustomWindowCommands/>
   </Fluent:RibbonWindow.WindowCommands>

    ....

```

but unexpectedly doing that resulted in _two sets_ of Minimize, Maximize, etc. buttons on the window!  I thought it would just replace the default `WindowCommands` object. I didn't see anything unusual in the definition of the `.WindowCommands` DP.

(Using the CustomWindowCommands class from code-behind is fine. )
